onnxruntime/core/providers/plaidml/plaidml_execution_provider.cc

```diff
@@ -122,32 +122,43 @@ common::Status PlaidMLExecutionProvider::Compile(
           Ort::CustomOpApi ort{*api};
           // TODO (PlaidML): why is this using custom op api? is it necessary ? is there a neater way?
           auto pml_state = static_cast<PlaidMLFunctionState*>(state);
-          auto binder = plaidml::exec::Binder(*pml_state->program->program);
-
+          pml_state->program->program->compile();
```

Comment:
Is there a target configuration that we should be using here that comes from ORT?